Project_19/Project_19/Views/Main.storyboard

```diff
@@ -162,7 +169,7 @@
             <objects>
                 <tableViewController extendedLayoutIncludesOpaqueBars="YES" id="F6c-Ug-Qpf" customClass="AddSetViewController" customModule="Project_19" customModuleProvider="target" sceneMemberID="viewController">
                     <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="static" style="insetGrouped" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="18" estimatedSectionHeaderHeight="-1" sectionFooterHeight="18" estimatedSectionFooterHeight="-1" id="UCl-Ed-83m">
-                        <rect key="frame" x="0.0" y="0.0" width="390" height="844"/>
+                        <rect key="frame" x="0.0" y="0.0" width="390" height="790"/>
```

Comment:
Let's try to make this more dynamic so it is not dependent on the phone screen size.